src/App.css

```diff
@@ -50,7 +50,7 @@
 }
 
 .task-name-container {
-  width: 640px;
+  width: 620px;
```

Comment:
In the design, the task name container is `640px`, maybe you accidentally change it to `620px`?